Base/Logic/Resources/CMakeLists.txt

```diff
@@ -18,3 +18,26 @@ install(
   DESTINATION ${Slicer_INSTALL_SHARE_DIR}/ColorFiles
   COMPONENT Runtime
   )
+
+# Related terminology files
+file(GLOB TERMINOLOGYLUTFILES RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "ColorFiles/Terminology/*.csv")
+file(GLOB REGIONANDCATEGORYFILES RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "ColorFiles/Terminology/*.xml")
```

Comment:
I would recommend not to glob files. If you do so, adding new files will not cause a reconfiguration and the file will not be collected into the variable `REGIONANDCATEGORYFILES` and `TERMINOLOGYLUTFILES`.

See https://cmake.org/cmake/help/git-next/command/file.html?highlight=glob 

Instead you could so something like this:

```
set(Slicer_TERMINOLOGY_DIR "ColorFiles/Terminology")

set(terminology_filenames

  # LUT
  GenericAnatomyColors-SNOMED.csv

  # Region and Category
  AnatomicRegionAndModifier.xml
  SegmentationCategoryTypeModifier.xml

  )
foreach(filename ${terminology_filenames})
  message(STATUS "Configuring terminology file: ${filename}")
  set(terminology_filepath ${CMAKE_CURRENT_SOURCE_DIR}/${Slicer_TERMINOLOGY_DIR}/${filename})
  configure_file(
    ${terminology_filepath}
    ${CMAKE_BINARY_DIR}/${Slicer_SHARE_DIR}/${Slicer_TERMINOLOGY_DIR}/${filename}
    COPYONLY
    )
  install(
    FILES ${terminology_filepath}
    DESTINATION ${Slicer_INSTALL_SHARE_DIR}/ColorFiles/${Slicer_TERMINOLOGY_DIR}
    COMPONENT Runtime
    )
endforeach()
```